HTWDD/Main Categories/General/AppDelegate.swift

```diff
@@ -43,6 +43,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         Tracker.track(.open)
     }
 
+    func application(_ application: UIApplication, open url: URL, sourceApplication: String?, annotation: Any) -> Bool {
+        
+        let viewController = url.host?.removingPercentEncoding
+        
+        if(viewController == "schedule") {
```

Comment:
`()` are not needed for if-statements in Swift.

- [ ] remove unnecessary syntax

Also: I think we should make this more generic. I don't have a good solution yet, but the scheme should maybe be capable of more routes than just `schedule`. But we could add this later as well 👍